public/css/module.less

```diff
@@ -490,6 +490,15 @@ td > a > .state-badges {
             width: 1.5em;
             height: 1.5em;
             border-radius: 0.3em;
+
+            &.handled-icon {
+                float: right;
+                pointer-events: none;
```

Comment:
If this wouldn't be an `a`, ignoring pointer events isn't required. Why not just add this class to the `i`?